src/OrbitGl/TimelineTicksTest.cpp

```diff
@@ -26,4 +26,28 @@ TEST(TimelineTicks, GetMajorTicks) {
                                    1 * kNanosecondsPerMinute));
 }
 
+static void CheckTicks(uint64_t start_ns, uint64_t end_ns, const std::set<uint64_t>& major_ticks,
+                       const std::set<uint64_t>& minor_ticks) {
+  TimelineTicks timeline_ticks;
+  auto all_ticks = timeline_ticks.GetAllTicks(start_ns, end_ns);
+  EXPECT_EQ(all_ticks.size(), major_ticks.size() + minor_ticks.size());
+  for (auto& [tick_type, timestamp_ns] : all_ticks) {
+    EXPECT_FALSE(tick_type == TimelineTicks::TickType::kMajorTick &&
+                 major_ticks.count(timestamp_ns) == 0);
+    EXPECT_FALSE(tick_type == TimelineTicks::TickType::kMinorTick &&
+                 minor_ticks.count(timestamp_ns) == 0);
```

Comment:
Nit: These should be 4 separate checks (EXPECT_EQ)

Reply:
Otherwise on a test failure gtest won't report which condition failed.

Reply:
Sorry, maybe I'm not being very clear. I can't refactor that into 4 EXPECT, because they are implicit implies
1- `If (is_minor_tick()) -> EXPECT_TRUE (minor_ticks.count())`
2-` If (is_major_tick()) -> EXPECT_TRUE (major_ticks.count())`
And each of them is checking that implicancy.

Actually, I can refactor it into only one EXPECT:    
```
EXPECT_TRUE(tick_type == TimelineTicks::TickType::kMajorTick
                    ? major_ticks.count(timestamp_ns) != 0
                    : minor_ticks.count(timestamp_ns) != 0);
```
but It's exactly what you are suggesting NOT to do.

Reply:
Yeah, you're right. I'm an idiot. It would only work for true. Sry....